flake.nix

```diff
@@ -881,8 +881,10 @@
                 cargo-udeps
                 pkgs.parallel
                 pkgs.semgrep
+                pkgs.just
 
                 (pkgs.writeShellScriptBin "git-recommit" "exec git commit --edit -F <(cat \"$(git rev-parse --git-path COMMIT_EDITMSG)\" | grep -v -E '^#.*') \"$@\"")
+                (pkgs.writeShellScriptBin "j" "exec just \"$@\"")
```

Comment:
unfortunately `j` conflicts with my [autojump](https://github.com/wting/autojump) but I guess that's *just* me :)

Reply:
Ha. I use zoxide and `z`. I will remove it. Anyone can add their own shortcut if they so desired. It was a bad idea.